cmd/istioctl/main.go

```diff
@@ -403,6 +407,58 @@ and destination policies.
 		},
 	}
 
+	configCmd = &cobra.Command{
+		Use:   "config-create <ip>:<port>",
+		Short: "Create base kubeconfig file for a server",
+		Example: `
+		# Create a config file for the api server.
+		istioctl config-create http://127.0.0.1:8080
+		`,
+		RunE: func(c *cobra.Command, args []string) error {
+
+			configName := "istio"
+
+			if len(args) < 1 {
+				c.Println(c.UsageString())
+				return fmt.Errorf("specify the the api server ip")
+			}
+
+			u, err := url.ParseRequestURI(args[0])
+			if err != nil {
+				c.Println(c.UsageString())
+				return err
+			}
+
+			configAccess := clientcmd.NewDefaultPathOptions()
+			// use specified kubeconfig file for the location of new config
+			configAccess.GlobalFile = kubeconfig
+
+			config, err := configAccess.GetStartingConfig()
```

Comment:
double checking. Why this kubeconfig thing in GlobalFile ? is that a fallback if we couldn't create a config ourselves? 
And does this function call fail if local config file is not present, but global one is present? I guess not.